packages/avatars/src/styled/StyledStatusIndicatorBase.ts

```diff
@@ -67,19 +67,20 @@ const sizeStyles = (props: IStyledStatusIndicatorProps) => {
   `;
 };
 
-const colorStyles = (props: IStyledStatusIndicatorProps) => {
-  let backgroundColor = getStatusColor(props.type, props.theme);
+const colorStyles = ({ theme, $type }: IStyledStatusIndicatorProps) => {
+  const foregroundColor = getColor({ variable: 'foreground.onEmphasis', theme });
+  let backgroundColor = getStatusColor(theme, $type);
   let borderColor = backgroundColor;
```

Comment:
```suggestion
  let backgroundColor;
  let borderColor;
```

...and move assignments to a conditional `else` clause below.

Reply:
This feels like a nit, but I guess I'm not seeing a gain in clarity with moving the assignment down a few lines when the variables can only be one of two values anyway?

Reply:
Could be a [nit], but in practice Garden tends to micro-optimize by deferring conditional assignment (and associated function call expense) until they are determined to be necessary. More like...

```
let backgroundColor;
let borderColor;

if ($type === 'offline') {
  backgroundColor = ...;
  borderColor = ...;
} else {
  backgroundColor = ...;
  borderColor = ...;
}
```

Reply:
I guess that's the nit I'm trying to understand. Whether the value is assigned at the initial assignment aka default value (`let thing = value`) or at the `else` (which is acting as the default value), this feels like a stylistic preference, right? I just want to make sure I understand that that's what we're talking about, because I'm not seeing any real optimization in this scenario. 🙂

Reply:
Okay, I think I understand now after looking at it for a minute. I didn't see initially that `getColor` would be called extraneously in current scenario. I'll update and push it up!